shell/plugins/steve/worker/web-worker.advanced.js

```diff
@@ -394,15 +484,79 @@ const workerActions = {
 
     Object.values(caches).forEach(c => c.setDebug(cache));
   },
-  updateBatch(type, id, change) {
+  singleUpdateBatch(type, id, change) {
     if (!state.batchChanges[type]) {
       state.batchChanges[type] = {};
     }
     state.batchChanges[type][id] = change;
   },
   sendBatch(requestHash) {
```

Comment:
this doesn't re-paginate (byPage) .. looks like this just pushes in-page resource updates